packages/nimble-components/src/accordion/specs/README.md

```diff
@@ -0,0 +1,137 @@
+    <nimble-accordion-item expanded>
+        <span slot="heading">Panel two</span>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <span slot="heading">Panel three</span>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+Complex accordion group with components in slots and single expand-mode
+
+```
+<nimble-accordion expand-mode="single">
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion one</div>
+        <nimble-select slot="end">
+            <nimble-option value="1">Option 1</nimble-option>
+            <nimble-option value="2">Option 2</nimble-option>
+            <nimble-option value="3">Option 3</nimble-option>
+        </nimble-select>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion two</div>
+        <nimble-number-field placeholder="number" slot="end"></nimble-number-field>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion three</div>
+        <nimble-select slot="end">
+            <nimble-option value="1">Option 1</nimble-option>
+            <nimble-option value="2">Option 2</nimble-option>
+            <nimble-option value="3">Option 3</nimble-option>
+        </nimble-select>
+    </nimble-accordion-item>
+    <nimble-accordion-item>
+        <nimble-checkbox slot="start"></nimble-checkbox>
+        <div slot="heading">Accordion four</div>
+        <nimble-number-field placeholder="number" slot="end"></nimble-number-field>
+    </nimble-accordion-item>
+</nimble-accordion>
+```
+
+### API
+
+#### Nimble Accordion
+
+[FAST accordion API](https://github.com/microsoft/fast/blob/57f3c22c6341d8a21d48b1ffb7fcbfab1ffd02d8/packages/web-components/fast-foundation/src/accordion/accordion.spec.md)
+
+-   _Component Name:_ `nimble-accordion`
+-   _Properties/Attributes:_
+    -   `appearance` - `block`, `outline`, `ghost`
+-   _Methods:_ Unchanged
+-   _Events:_ Unchanged
+-   _CSS Classes and Custom Properties that affect the component:_ Unchanged
+-   _Slots:_ Unchanged
+
+The Figma design includes appearances of the accordion header that reflect those of the `block`, `outline`, and `ghost` appearances used in other components. Because of the difference in border functionality and possibility of excess overridden css classes if patterns are used, shared styles will not be used, and new styles for the accordion will be created. The appearances of `block`, `outline`, and `ghost` will be implemented under the HTML attribute `appearance`, which will use conditional css styles based on `appearance`'s value (ex. :host([appearance='outline']) will have styling for the outline button).
+
+Documentation will be added to advise against using multiple Nimble Accordions with different appearances next to each other.
```

Comment:
It'd be good to check with Brandon if he has guidance about when to use each `appearance`. Is it unique guidance for the accordion or can we re-use some guidance from the button? (This needn't block this PR but you could add a note mentioning it)

Reply:
Sure, I'll add that along with the other comments in the visual design section and I'll ask him when he gets back 👍